erpnext_documentation/www/docs/user/manual/en/support/issue.md

```diff
@@ -58,10 +61,14 @@ User can select the [Service Level Agreement](/docs/user/manual/en/support/servi
 <img class="screenshot" alt="Issue" src="{{docs_base_url}}/assets/img/support/iss.gif">
 
 ### 3.3 Response
-**Mins to First Response**: Time in minutes from when the Issue was created to when the first reply was sent.
+* **Mins to First Response**: Time in minutes from when the Issue was created to when the first reply was sent.
 
 * **First Responded on**: When a Support team member first replies to the issue, first response date and time will be updated.
 
+* **Average Response Time**: The average time taken to respond to the Customer. This is calculated as the average of all time spans between Received and Sent Communications. This field will be updated on every reply to the customer.
```

Comment:
This is the average time between all sent and received communications

Reply:
Actually it is specifically "Response Time" so it is better if the definition is in detail

Reply:
The average time taken to respond to the Customer. This is calculated by taking the average of all time spans between Received and Sent Communications. This field will be updated on every reply to the customer.